build/transpile.js

```diff
@@ -2430,9 +2431,7 @@ class Transpiler {
             pyAsync: [
                 "import asyncio",
                 "import ccxt.async_support as ccxt  # noqa: E402",
-                "",
```

Comment:
all the changes inside `transpile.js` is done within `transpileExamples` method, so no other areas are affected.